device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -1450,15 +1459,7 @@ private void handleMessageException(IotHubTransportPacket packet, TransportExcep
             log.warn("The device operation timeout has been exceeded for the message, so it has been abandoned ({})", packet.getMessage(), transportException);
         }
 
-        IotHubStatusCode errorCode = (transportException instanceof IotHubServiceException) ?
```

Comment:
A more exhaustive version of this transportException to status code mapping exists elsewhere, so removing this incomplete mapping